midealocal/cloud.py

```diff
@@ -126,8 +126,13 @@ def _redact_data(data: str) -> str:
         + getattr(cr, "street_addresses", [])
     )
     for token in token_list:
-        m = len(token)
-        elm = r"\b" + token + r"\b"
+        item = token
```

Comment:
Why not using token directly ?

Reply:
to identify the begining and end of each token